packages/anoncreds/src/models/exchange.ts

```diff
@@ -102,9 +102,9 @@ export interface AnonCredsProofRequest {
       p_type: '>=' | '>' | '<=' | '<'
       p_value: number
       restrictions?: WalletQuery[]
```

Comment:
Not from your PR but we should probably rename the `WalletQuery` to something else.

Reply:
Yeah probably something like `AnonCredsProofRequestRestriction`